test/useFind.test.ts

```diff
@@ -175,6 +185,82 @@ describe('Find composition', () => {
     );
   });
 
+  it('should un-load data after params changes to ref with undefined or null', async () => {
+    expect.assertions(4);
+
+    // given
+    const findParams = ref<Params | undefined>({ query: { zug: 'start' } });
+    const serviceFind = jest.fn(() => Promise.resolve([testModel]));
+    const feathersMock = {
+      service: () => ({
+        find: serviceFind,
+        on: jest.fn(),
+        off: jest.fn(),
+      }),
+      on: jest.fn(),
+      off: jest.fn(),
+    } as unknown as Application;
+    const useFind = useFindOriginal(feathersMock);
+    let findComposition = null as UseFind<TestModel> | null;
+    mountComposition(() => {
+      findComposition = useFind('testModels', findParams);
+    });
+
+    // before then to ensure that the previous loading procedure is completed
+    await nextTick();
+    expect(findComposition && findComposition.data.value).toStrictEqual([testModel]);
+
+    // when
+    findParams.value = undefined;
```

Comment:
The test title mentions `null` but this only checks with `undefined`. :thinking:

Reply:
@anbraten How should we move forward with this. I would suggest to choose either `null` or `undefined` and not allow both.

Reply:
@lukashass Okay so we have `useFind('service')` => no params at all which will use default and query all objects and `useFind('service', computed(() => undefined))` which should stop the search and the case with an object as param?

Reply:
We will allow `undefined | null` since we allowed that in `useGet` as well. 

https://github.com/geprog/use-feathers/blob/2c996de28681155b709001834724cdd5c96c27ed/src/useGet.ts#L67